packages/cli/generators/project/templates/.mocharc.js

```diff
@@ -0,0 +1,9 @@
+const semver = require('semver');
```

Comment:
Please note this code is going to be run with generated LoopBack applications. The templates for `package.json` does not have `semver` in `devDependencies` yet. It will be problematic.

We also use `source-map-support` in two other places:
- https://github.com/strongloop/loopback-next/blob/master/packages/cli/generators/project/templates/package.json.ejs#L101
- https://github.com/strongloop/loopback-next/blob/master/packages/cli/generators/project/templates/package.json.ejs#L147

Ideally, we need to make them conditional based on the node.js version too. 

Maybe it's not ready for us to update the code generation at this moment for the following issues I'm seeing:
1. A generated LB app can be possibly run with Node 10/12/14. Using `source-map-support` for now is probably the best.
2. It will be difficult to assert snapshots for the generated code which are different between node.js versions.

Reply:
so you prefer to wait for node v12 to be the minimum version of loopback and not conditionally polyfill this?

Reply:
Personally, I would prefer to wait for Node.js 12. I just feel it's not worth the effort that creates a complex solution with not much gain. I'll let @bajtos  to weigh in.

Reply:
I don't have a strong opinion. On one hand, I see the benefits of using built-in source map support - for example, it prints information about both TS and JS line location, which makes it easier to troubleshoot errors caused by specific output of TypeScript compiler. On the other hand, I also agree the currently proposed solution is adding quite a lot of complexity, which may not justify the benefits.

Reply:
Maybe the best compromise is to clean up the PR to use Node.js built-in source map support and wait until Node 10 is EOL to land it.